### DISCUSSION TITLE
Welcome to ApkEditor Discussions!

### DISCUSSION THREAD

**Discussion by PatrickAlex2019** [PatrickAlex2019/ApkEditor#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by KIDU333** [comment]
Any one help me i wont to creat patch file for apk editor pro what do i need  the patch i wont to creat is a simple log in page

**Comment by MDrzd** [comment]
May I get the source code of ApkEditor?

**Comment by Gricecal** [comment]
I just want to thank you for making this amazing app!! Its the only app I can get to clone apks successfully on android without using a computer of course. All I have to do to clone it is change the app name and app package name. ❤️ I cannot get apk cloners to work from google play. I tried many of them and none worked for me. I don't know why either.
